arch/arm/Kconfig

```diff
@@ -901,6 +901,12 @@ config ARM_HAVE_NEON
 	---help---
 		Decide whether support NEON instruction
 
+config ARM_HAVE_MVE
```

Comment:
Do we have any chip in-tree that has `ARM_HAVE_MVE=y` or this is reserved for future?

Reply:
Our partner has chip with MVE, but not upstream yet. Since MVE is ARM standard IP, it expect more chip will ship to market soon:
https://developer.arm.com/architectures/instruction-sets/simd-isas/helium